### PR TITLE
Checking _Rendezvous.done() when stopping Pub / Sub request generator.

### PR DESCRIPTION
**Pull request by dhermes**

### CODE REVIEW

File: pubsub/google/cloud/pubsub_v1/subscriber/_consumer.py [hunk 1]
```diff
@@ -237,12 +237,23 @@ def _stop_request_generator(self, request_generator):
         Args:
             request_generator (Generator): A streaming pull request generator
                 returned from :meth:`_request_generator_thread`.
+            response_generator (grpc._channel._Rendezvous): The gRPC
```
Comment (jonparrott) [inline pubsub/google/cloud/pubsub_v1/subscriber/_consumer.py]: The type you're looking for is `grpc.Call`.

Comment (dhermes) [reply pubsub/google/cloud/pubsub_v1/subscriber/_consumer.py]: I can change it if you want, but I feel like that's me yielding to the insanity that is `_Rendezvous`, so I am not keen on it.

Comment (jonparrott) [reply pubsub/google/cloud/pubsub_v1/subscriber/_consumer.py]: I'd really prefer it, and actually, to add to the madness this is actually [`grpc.Future`](https://grpc.io/grpc/python/grpc.html#grpc.Future).

**Comment by dhermes** [comment]
Updated docstring (no code changes). Merging without CI (since no code changes).

**Pull request merged**